### DISCUSSION TITLE
Random seed for RandomLinkSplit class

### DISCUSSION THREAD

**Discussion by YiheWang** [pyg-team/pytorch_geometric#5534] [Q&A] [answered]
Hi, is there any way to fix the random seed for RandomLinkSplit class? I want to keep my training, val, and test set the same each time splitting.

**Comment by rusty1s** [comment, marked as answer]
You can do `torch_geometric.seed_everything(12345)`.

**Reply by YiheWang** [reply]
Thanks!
